ci-operator/config/jws-qe/interop-ocp-ci/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+- cspi-qe-ocp-lp
```

Comment:
Do any of the JWS PQE members want to have permissions for changes?

Reply:
for this they need to have github user right with access to release repo?
Since as a team they are not using github so am not sure at the moment it would be possible to add them

Reply:
I'll add it later if I get a response from PQE

Reply:
Sounds good to me! They can follow this guide if they want to: https://source.redhat.com/groups/public/openshift/openshift_wiki/openshift_onboarding_checklist_for_github